### PR TITLE
Fix TEST SQLALCHEMY_DATABASE_URI for tests

### PR DESCRIPTION
**Pull request by qqlexa**

### CODE REVIEW

**Review by RezenkovD — CHANGES_REQUESTED** [review]

File: src/app/routers/statistics.py [hunk 1]
```diff
@@ -29,23 +30,23 @@ def transform_date_or_422(date_: str) -> date:
     return transformed_date
 
 
-@router.get("/{user_id}/statistics", response_model=src.schemas.statistics.Statistics)
+@router.get("/{user_id}/statistics", response_model=schemas.Statistics)
 def get_stats(user_id: int, db: Session = Depends(get_db)):
```
[the remark this answers is not in the document]
Comment (qqlexa) [reply src/app/routers/statistics.py:34]: You are right. Let's change it in the future (not a requirement for this MR).

File: tests/conftest.py [hunk 2]
```diff
@@ -4,7 +4,24 @@
...
 from src.db.database import get_db
 from src.db.database import Base
-from src.db.test_database import engine, override_get_db
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from src.config import settings
+
+SQLALCHEMY_DATABASE_URI = settings.SQLALCHEMY_DATABASE_URI
```
[the remark this answers is not in the document]
Comment (qqlexa) [reply tests/conftest.py:12]: Changed

File: src/app/routers/statistics.py [hunk 3]
```diff
@@ -29,23 +30,23 @@ def transform_date_or_422(date_: str) -> date:
     return transformed_date
 
 
-@router.get("/{user_id}/statistics", response_model=src.schemas.statistics.Statistics)
+@router.get("/{user_id}/statistics", response_model=schemas.Statistics)
```
[the remark this answers is not in the document]
Comment (qqlexa) [reply src/app/routers/statistics.py:33]: Duplicated comment

File: .gitignore [hunk 4]
```diff
@@ -1,2 +1,9 @@
 .idea
 __pycache__/
+
+# To ignore result of tests
```
Comment (RezenkovD) [inline .gitignore:4]: Why do we need _coverage_ , etc?

**Review by RezenkovD — CHANGES_REQUESTED** [review]

**Review by RezenkovD — APPROVED** [review]

**Pull request merged**